coordinatorlib/src/main/scala/com/socrata/datacoordinator/secondary/SecondaryInfo.scala

```diff
@@ -1,11 +1,31 @@
 package com.socrata.datacoordinator.secondary
 
+import java.sql.ResultSet
+
 import org.joda.time.DateTime
 
-case class SecondaryConfigInfo(storeId: String, nextRunTime: DateTime, runIntervalSeconds: Int)
+case class SecondaryInstanceInfo(storeId: String, nextRunTime: DateTime, runIntervalSeconds: Int)
+object SecondaryInstanceInfo {
+  def apply(rs: ResultSet): SecondaryInstanceInfo =
+    SecondaryInstanceInfo(rs.getString("store_id"),
+      new DateTime(rs.getTimestamp("next_run_time").getTime),
+      rs.getInt("interval_in_seconds"))
+}
+
+case class SecondaryGroupInfo(groupId: String, isDefault: Boolean, instances: Set[String], numReplicas: Int)
+object SecondaryGroupInfo {
+  def apply(rs: ResultSet): SecondaryGroupInfo =
+    SecondaryGroupInfo(rs.getString("id"),
+      rs.getBoolean("is_default"),
+      rs.getString("instances").split("[ ,;]").toSet.map((s: String) => s.trim),
```

Comment:
Can we actually model this in the database in an array or something?  Feels kind of dirty saying "oh, this is a string using whatever delimiters we want to throw in".